src/polyfills/document-contains.js

```diff
@@ -26,8 +26,11 @@ function documentContainsPolyfill(node) {
   // Per spec, "contains" method is inclusionary
   // i.e. `node.contains(node) == true`. However, we still need to test
   // equality to the document itself.
-  // eslint-disable-next-line local/no-invalid-this
-  return node == this || this.documentElement.contains(node);
+
+  return (
+    node == this || // eslint-disable-line local/no-invalid-this
+    /**@type {!Document} */ (this).documentElement.contains(node) // eslint-disable-line local/no-invalid-this
```

Comment:
Shouldn't this just take the place of the `@this` annotation above?

Reply:
Great suggestion! DOne